app/event/send-route-event.js

```diff
@@ -0,0 +1,49 @@
+const raiseEvent = require('./raise-event')
+const config = require('../config')
+const { EventPublisher } = require('ffc-pay-event-publisher')
+const { SOURCE } = require('../constants/source')
+const { PAYMENT_PAUSED_PREFIX } = require('../constants/events')
+
+const sendProcessingRouteEvent = async (paymentRequest, routeLocation, routeType) => {
+  if (config.useV1Events) {
+    await sendV1ProcessingRouteEvent(paymentRequest, routeLocation, routeType)
+  }
+  if (config.useV2Events) {
+    await sendV2ProcessingRouteEvent(paymentRequest, routeLocation, routeType)
+  }
+}
+
+const sendV1ProcessingRouteEvent = async (paymentRequest, routeLocation, routeType) => {
+  const direction = routeType === 'request' ? 'to' : 'from'
+  const event = {
+    id: paymentRequest.correlationId,
+    name: `payment-request-${routeLocation}-${routeType}`,
+    type: 'info',
+    message: `Payment request routed ${direction} request editor`,
+    data: paymentRequest
+  }
+  await raiseEvent(event)
+}
+
+const sendV2ProcessingRouteEvent = async (paymentRequest, routeLocation, routeType) => {
+  if (routeType === 'request') {
```

Comment:
what if `routeType` is !== 'request' ? This would do nothing, is that the intended functionality here? 
Is there no need for a `direction` variable as on line 17 above for the V1 events

Reply:
Yes, that is intended.  Sending an event on response adds no value as it's duplicated with events emitted from request editor.

Once we remove V1 events we can get rid of direction